apps/react-storybook/stories/scheduler/SchedulerFormCustomization.stories.tsx

```diff
@@ -332,21 +335,109 @@ export const RTL: Story = {
 export const LegacyPopup: Story = {
   args: {
     ...baseConfig,
-    "editing.form.iconsShowMode": "none",
+    "editing.form.iconsShowMode": "both",
   } as Properties,
   argTypes: iconsShowModeArgType,
-  render: (args) => {
+  render: () => {
+    let form: InstanceType<typeof dxForm> | null = null;
+
+    const schedulerRef = useRef(null);
+
     return (
-      <Scheduler
+      // @ts-ignore
+      <ReactScheduler
+        ref={schedulerRef}
         {...baseConfig}
         editing={{
+          popup: {
+            maxWidth: 800,
+            toolbarItems: [
+              {
+                toolbar: 'top',
+                location: 'before',
+                text: "Edit Appointment",
+                cssClass: 'dx-toolbar-label',
+              },
+              {
+                toolbar: 'top',
+                location: 'after',
+                options: {
+                  stylingMode: 'contained',
+                  type: 'default',
+                  onClick: () => {
+                    // @ts-ignore
+                    schedulerRef.current?.instance()?.hideAppointmentPopup(true);
+                  },
+                  text: 'Save',
+                },
+                shortcut: 'done',
+              },
+              {
+                toolbar: 'top',
+                location: 'after',
+                shortcut: 'cancel',
+              },
+            ],
+          },
           form: {
+            onContentReady: function (e) {
+              form = e.component;
+            },
+            iconsShowMode: "both",
             items: [
-              "mainGroup",
-              {name: 'recurenceGroup', visible: true}
+              {
+                name: "mainGroup",
+                cssClass: "",
+                items: [
+                  "subjectGroup",
+                  "dateGroup",
+                  {
+                    name: "repeatGroup",
+                    items: [
+                      "repeatIcon",
+                      {
+                        name: "customRepeatEditor",
+                        editorType: "dxSwitch",
+                        dataField: "repeat",
+                        editorOptions: {
+                          onValueChanged: (e) => {
+                            if (e.value === true) {
+                              form?.option("colCount", 2);
+
+                              const recurrenceRule = form?.option('formData')?.recurrenceRule;
+
+                              form?.option('formData', {
```

Comment:
Minor:
It's possible to access recurrence rule by: `form.option(formData.recurrenceRule)`.

So we can replace it to:
form?.option('formData.recurrenceRule', recurrenceRule || "FREQ=DAILY");